docs/src/guides/strapi/deploy/configure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Configure Strapi for Platform.sh"
+sidebarTitle: "Configure"
+weight: -100
+toc: false
+description: |
+  Review the basics of what makes up a Platform.sh project, including its three principle configuration files and how to define them for Strapi.
+---
+
+{{< guides/config-desc name="Strapi" >}}
+
+## Requests configuration: `routes.yaml`
+
+{{< guides/config-routes template="gatsby" name="Strapi" >}}
+
+## Service configuration: `services.yaml`
+
+The `services.yaml` file lists the pre-packaged services you need for your application to run. You pick the major version of the service, and Platform.sh updates the patch version periodically so that you always get the newest version when you deploy.
+
+Strapi requires a database to be able to deploy, by default it uses an SQLlite database but other database types are also supported.
+These other database types are Oracle MySQL, PostgreSQL or MonogoDB(available only in strapi v3 and below). In the strapi template, a postgres database service is defined. So you can add the following to your `services.yaml` file:
+
+```yaml
+# The services of the project.
+
+# Each service listed will be deployed
+# to power your Platform.sh project.
+
+dbpostgres:
+  type: postgresql:12
+  disk: 256
+# Uncomment the line below if you would like to use a mysql database
+# dbmysql:
+#   type: oracle-mysql:8.0
+#   disk: 256
+```
+
+Apart from databases, you can add other services if desired, such as [Solr](/configuration/services/solr.md) or [Elasticsearch](/configuration/services/elasticsearch.md). You will need to configure Strapi to use those services as well once the service is enabled.
+
+## Application container: `.platform.app.yaml`
+
+{{< guides/config-app template="strapi" >}}
```

Comment:
```
    W: Error: Error building site: "/app/src/guides/strapi/deploy/configure.md:42:1": failed to render shortcode "guides/config-app": failed to process shortcode: "/app/layouts/shortcodes/guides/config-app.html:7:15": execute of template failed: template: shortcodes/guides/config-app.html:7:15: executing "shortcodes/guides/config-app.html" at <readFile $file>: error calling readFile: stat /app/static/files/fetch/appyaml/strapi: no such file or directory
```

For this, you'll have to add the template name to this file https://github.com/platformsh/platformsh-docs/blob/main/docs/data/remote-examples/templates/app.yaml so that it's actually available during the docs build. I would think we should reference `strapi4` here as well, no?